scope.Rmd

```diff
@@ -143,29 +144,84 @@ other contexts.
+regardless of categories. General standards may simply require that a piece of
+software is demonstrably superior to otherwise equivalent software in at least
+one (or perhaps more?) specific way(s). Submissions may occasionally be truly
+unique, and therefore devoid of any "otherwise equivalent software". It will
+nevertheless be expected that otherwise equivalent software will generally
+exist, in which cases such standards might accordingly be approached through
+considering the following kinds of questions, made in regard to one or more
+"reference implementations" of otherwise equivalent software, and which
+software authors may be requested to specify:
+
+- [ ] **Efficiency:** Is the software more efficient (faster, simpler, other
+  interpretations of "efficient") than reference implementations?
+- [ ] **Reproducibility or Reliability:** Does the software reproduce
+  sufficiently similar results more frequently than reference implementations
+  (or otherwise satisfy similar interpretations of reproducibility)?
+- [ ] **Accuracy or Precision:** Is the software demonstrably more accurate or
+  precise than reference implementations (such as ?
+- [ ] **Simplicity of Use:** Is the software simpler to use than reference
+  implementations?
+- [ ] **Algorithmic Characteristics:** Does the algorithmic implementation
+  offer characteristics (such as greater simplicity or sensitivity) superior to
+  reference implementations? If so, which?
+- [ ] **Convergence:** Does the software provide faster or otherwise better
+  convergence properties than reference implementations?
+- [ ] **Method Validity:** Does the software overcome demonstrable flaws in
+  previous (reference) implementations? If so, how?
+- [ ] **Method Appliciability:** Does the software enable a statistical method
+  to be applied to a domain in which such application was not previously
+  possible?
+- [ ] **Automation** Does the software automate aspects of statistical analyses
+  which previously (in a reference implementation) required manual intervention?
+- [ ] **Input Data:** Does the software "open up" a method to input data
+  previously unable to be treated by a particular algorithm or method?
+- [ ] **Output Data:** Does the software provide output in forms previously
+  unavailable by reference implementations?
+- [ ] **Reference Standards:** Are there any reference standards, such as the
+  US National Institute of Standards and Technology's
+  [  collection of reference data sets](https://www.itl.nist.gov/div898/strd)
+  against which the software may be compared? If so, which?
+
+***Proposal*** Submissions will be required to check at least one of the
+preceding items, to nominate at least one "reference implementation", and to
+explain how the submitted software is superior (along with a possibility to
+explain why software may be sufficiently unique that there is no reference
+implementation, and so no claims of superiority can be made).
+
```

Comment:
I think this section is very good and comprehensive, but I've figured out that I think it  t belongs in the _next_ discussion/section when we talk about the review flow and things like whether we want binary or graded rating, one-time or ongoing evaluation criteria, etc. I'd suggest cutting it and moving it to the next chapter.